1.split_data/scripts/nbconverted/split_data.py

```diff
@@ -25,27 +24,29 @@
 
 
 # load x (features) and y (labels) dataframes
-load_path = pathlib.Path("../0.download_data/data/training_data.csv.gz")
-training_data = get_features_data(load_path)
-print(training_data.shape)
-
-# ratio of data to be reserved for testing (ex 0.15 = 15%)
-test_ratio = 0.15
+labeled_data_path = pathlib.Path("../0.download_data/data/labeled_data.csv.gz")
+labeled_data = get_features_data(labeled_data_path)
+print(labeled_data.shape)
 
 
 # In[3]:
 
 
+# ratio of data to be reserved for testing (ex 0.15 = 15%)
+test_ratio = 0.15
+
 # test_data is pandas dataframe with test split, stratified by Mitocheck_Phenotypic_Class
-test_data = training_data.groupby("Mitocheck_Phenotypic_Class", group_keys=False).apply(
+testing_data = labeled_data.groupby("Mitocheck_Phenotypic_Class", group_keys=False).apply(
```

Comment:
you probably should be setting a seed at the beginning of this notebook (sorry if you are already!)

Also, why not use sklearn.model_selection.train_test_split? https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html